src/Cake.AzureDevOps/Pipelines/AzureDevOpsBuild.cs

```diff
@@ -6,6 +6,7 @@
     using Cake.AzureDevOps.Authentication;
     using Cake.Core.Diagnostics;
     using Microsoft.TeamFoundation.Build.WebApi;
+    using Microsoft.TeamFoundation.TestManagement.WebApi;
```

Comment:
Have you tested the addin already if it works with the assemblies which we ship with it? Because we only package/ship the assemblies which are actually used by the addin and not all dependencies.

Reply:
Added a reference to Microsoft.TeamFoundation.TestManagement.WebApi in the project file.